_posts/2017-11-07-amp-roadshow.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title:  "AMP Roadshow in Munich"
+date: 2017-11-07 23:03:00 +0200
+categories:  
+author_name: Wolfram Kriesing
+author_url : /author/wolframkriesing
+author_avatar: wolframkriesing
+show_avatar : true
+read_time : 7
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-11-amp-roadshow/road.jpg
+---
+
+Google invited [lots of developers][devs-tweet] to the [AMP Roadshow][amp-roadshow] that they are currently doing worldwide.
+We joined the Munich edition in their offices. 
+Google does that to promote their technology [AMP][amp]. Now it was on us to figure out
+what value could AMP add for HolidayCheck.
+I personally only knew the 10.000 feet view of AMP, which basically is what the abbreviation 
+stands for: "Accelerated Mobile Pages". But I learned quickly that it was not just about mobile.
+It was about fast web sites that are also responsive, serving any device's dimension.
+
+[devs-tweet]: https://twitter.com/wolframkriesing/status/927820303270662144
+[amp-roadshow]: https://www.ampproject.org/amp-roadshow/
+[amp]: https://www.ampproject.org/
+
+## Intro
+
+When I arrived at 9:00 and saw already about twenty people waiting and I thought I was late. But actually the event didn't start until 9:45 with about 120 people. Many people from the huge Munich community showed up. Maybe the Google food was attracting the people :). Ah no, AMP was the topic. So it was rather topics like SEO, ads, fast websites and the expected profit for ones business that made the people come. I guess.
+
+<div style="float: left; padding: 1rem;">
+<img src="{{site.baseurl}}/img/posts/2017-11-amp-roadshow/1-intro.jpg" alt="some numbers" width="100" class="sizeup-onhover-image scale3 origin-left-top" />
+<br/><em>Some numbers</em>
+</div>
+
+The official part was opened by a short five minutes "hello" and sales intro.
+Later [Paul Bakaus][paul], developer evangelist for the AMP team, started the day by setting the stage for AMP, showing some numbers and aligning everyone on the problem that AMP tries to solve: fast websites. It was said that there are already about 4 billion AMP pages out there and 90 new ones pop up every second, that was impressive to hear. Supporting the AMP story and state the well-known facts that decreasing a page load times improve conversion rates made everyone want to know more. What caught my attention was the statement that AMP is just a library. Now you got my ear, I am a developer and a library sounds usable to me.
```

Comment:
Some hints on this section:
- Maybe rather `He mentioned...` than `It was said` (less passive).
- `every second, that was ...` -> `every second. That was ...`
- `... and state the ...` -> `.. and stating the ..`